example/src/components/P12Inputs.tsx

```diff
@@ -51,11 +52,25 @@ const P12Inputs = (props: P12InputsProps) => {
         }
     };
 
+    const attachCertificate = async () => {
+        const fromUrl =
+            "https://github.com/mattermost/react-native-network-client/raw/e2e-cert/example/certs/client_cert.p12";
```

Comment:
I will update this url later to `https://github.com/mattermost/react-native-network-client/raw/master/example/certs/client_cert.p12` once merged to master